docs/tutorials/find-tasks-by-user-id.md

```diff
@@ -0,0 +1,67 @@
+# Find tasks by user ID
+
+In this tutorial, you learn the operations to call to find tasks by auto-generated user ID.
+
+Expect this tutorial to take about 15 minutes to complete.
+
+# Before you start
+
+1. Make sure you've completed the [Before-you-start-a-tutorial] topic on the development system you'll use for the tutorial.
+1. Check whether the {{base_url}} in Postman collection points to http://localhost:3000/. If not set correctly, you will see local host errors. For ****assistance setting up the variables for your import collection, refer to [set environment variables in Postman](https://learning.postman.com/docs/sending-requests/variables/environment-variables/).  
+
+
+# About this task
+
+You can retrieve a task or multiple tasks by a specific user ID.
+
+Steps to find tasks by user ID:
+
+1. Open your Postman desktop application. 
+1. Go to the **To-Do-mock** service test json  collection containing the list of users and tasks associated with each user. 
+1. Expand the **tasks** folder.
+1. Expand the **tasks:READ** method.
+1. From the list, select *Get tasks by user_id* and click on the three dots.
+1. From the options, select **Duplicate** and make a copy of the *Get tasks by user_id* request.
+1. (**Optional**) Make sure to select the **GET** method type. 
+1. (**Optional**) Rename the request. Here's an example: **Get tasks by user_id_2**. 
+1. Save the new request.
+1. Specify an user ID to the base URL request. For example: http://localhost:3000/tasks?user_id=1. Alternatively, you can modify the **Query Params** tab and then add a user ID. 
+1. Click **Send** to initiate your new request.  
+
+| Property  | Type | Description|
+|---|---|---|
+| User_id  | Integer   | (**Required**) Indicates a numeric value associated with each task a specific user owns.|
+
```

Comment:
I'm not sure what the preceding table refers to. It looks like you've finished the procedure already.

Reply:
How do we code tables between steps in Markdown? 

I wanted to keep table after step 10, so the user is aware of the accepted input type. But keeping that table after step 10 resulted step 11 to restart numbering to step 1. So I moved that table out of the procedure.

Reply:
You need to indent the table to be 1 level more indented than the procedure step you want it to be in.